starkinfra/dynamicbrcode/__dynamicbrcode.py

```diff
@@ -0,0 +1,112 @@
+
+_resource = {"class": DynamicBrcode, "name": "DynamicBrcode"}
+
+
+def create(brcodes, user=None):
+    """# Create DynamicBrcodes
+    Send a list of DynamicBrcode objects for creation at the Stark Infra API
+    ## Parameters (required):
+    - dynamicBrcode [list of DynamicBrcode objects]: DynamicBrcode objects to be created in the API.
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - list of DynamicBrcode objects with updated attributes
+    """
+    return rest.post_multi(resource=_resource, entities=brcodes, user=user)
+
+
+def get(uuid, user=None):
+    """# Retrieve a specific DynamicBrcode
+    Receive a single DynamicBrcode object previously created in the Stark Infra API by its uuid
+    ## Parameters (required):
+    - uuid [string]: object unique uuid. ex: "901e71f2447c43c886f58366a5432c4b"
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - DynamicBrcode object with updated attributes
+    """
+    return rest.get_id(resource=_resource, id=uuid, user=user)
+
+
+def query(limit=None, after=None, before=None, uuids=None, user=None):
+    """# Retrieve DynamicBrcodes
+    Receive a generator of DynamicBrcode objects previously created in the Stark Infra API
+    ## Parameters (optional):
+    - limit [integer, default None]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
+    - after [datetime.date or string, default None] date filter for objects created only after specified date. ex: datetime.date(2020, 3, 10)
+    - before [datetime.date or string, default None] date filter for objects created only before specified date. ex: datetime.date(2020, 3, 10)
+    - uuids [list of strings, default None]: list of uuids to filter retrieved objects. ex: ["901e71f2447c43c886f58366a5432c4b", "4e2eab725ddd495f9c98ffd97440702d"]
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - generator of DynamicBrcode objects with updated attributes
+    """
+    return rest.get_stream(
+        resource=_resource,
+        limit=limit,
+        after=check_date(after),
+        before=check_date(before),
+        uuids=uuids,
+        user=user,
+    )
+
+
+def page(cursor=None, limit=None, after=None, before=None, uuids=None, user=None):
+    """# Retrieve DynamicBrcodes
+    Receive a list of DynamicBrcode objects previously created in the Stark Infra API and the cursor to the next page.
+    ## Parameters (optional):
+    - cursor [string, default None]: cursor returned on the previous page function call
+    - limit [integer, default None]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
```

Comment:
para o page, o limite é diferente, o máximo é 100. 
`- limit [integer, default 100]: maximum number of objects to be retrieved. Max = 100. ex: 35`